mozilla_repo_urls/errors.py

```diff
@@ -8,8 +8,13 @@ def __init__(self, url_string) -> None:
 
 
 class UnsupportedPlatformError(RepoUrlsBaseError):
+    @property
+    def supported_platforms(self):
+        return list(self._supported_platforms)
+
     def __init__(self, url_string, platform, supported_platforms) -> None:
+        self._supported_platforms = supported_platforms
```

Comment:
From previous comment: if my understanding is correct, we can just make that attribute public.

```suggestion
        self.supported_hosts = supported_hosts
```

Reply:
Circling back after reviewing https://github.com/taskcluster/taskgraph/pull/126#pullrequestreview-1140599675, we likely don't need this attribute anymore.